src/pages/MetricDetail.svelte

```diff
@@ -113,132 +115,41 @@
     in
     <a href={`/apps/${params.app}`}>{params.app}</a>
```

Comment:
Should we include the app name here? I feel it's a bit redundant as the user can see the app name right on top of this with the breadcrumb.

Reply:
I think you're right, let's take out the app name.